packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,33 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```sh
+npm install
+```
+### Run
+To start the project is as easy as this
+```sh
+npm run start
+```
+
+### Documentation
```

Comment:
I'm thinking whether we can keep it brief on the description. Borrowing the template I got from a react app (https://github.com/dhmlau/dashboard-web), my proposal changes:

```
### Learn More
You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/).
The developer guide can be found: https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md. 
```

What do you think?

Reply:
+1 I think it's better to keep it brief

Reply:
Great!